test/app/views/integration_examples/layout.html.erb

```diff
@@ -0,0 +1,2 @@
+layout:<%= render partial: "test_layout" %>
+component:<%= render LayoutComponent %>
```

Comment:
Might we pass block content to this component and the partial render call above to ensure the two behaviors are the same?